### DISCUSSION TITLE
MacOS10.13.6

### DISCUSSION THREAD

**Discussion by lilian18** [upscayl/upscayl#554] [Q&A] [answered]
Hi, is there a version of Upscayl that’s compatable with MacOS10.13.6? thanks!

**Comment by aaronliu0130** [comment, marked as answer]
Hi! This is because of #98.

> This is because GitHub has removed access to anything other than MacOS 11+. So, sadly we won't be supporting MacOS 10.x going forward (Not unless [NayamAmarshe] buy a used MacBook or something).
